README.md

```diff
@@ -54,6 +54,38 @@ import * as Turbo from '@hotwired/turbo'
 +TurboPower.initialize(Turbo.StreamActions)
 ```
 
+### Installation on a stock Rails 7 install with importmaps
+
+1. `./bin/importmap pin turbo_power`
+
+2. You'll then have to make sure the following files are modified as such:
+
+**config/importmaps.rb**
+```diff
+pin "application", preload: true
+- pin "@hotwired/turbo-rails", to: "turbo.min.js", preload: true
++ pin "@hotwired/turbo", to: "turbo.min.js", preload: true
+pin "@hotwired/stimulus", to: "stimulus.min.js", preload: true
+pin "@hotwired/stimulus-loading", to: "stimulus-loading.js", preload: true
+- pin "@hotwired/turbo", to: "https://ga.jspm.io/npm:@hotwired/turbo@7.2.0/dist/turbo.es2017-esm.js"
+pin "turbo_power", to: "https://ga.jspm.io/npm:turbo_power@0.1.4/dist/index.js"
++ // or use the bundled script shipped with the `turbo_power` gem
++ pin "turbo_power", to: "turbo_power.js"
+```
+
+**app/javascripts/application.js**
+```diff
+- import '@hotwired/turbo-rails'
++ import { Turbo } from '@hotwired/turbo'
+import '@rails/request.js'
```

Comment:
Can we remove this one since it's not part of the default stack?
```suggestion
```

Reply:
Right – done.